analytics/policy/services/business_licence_client.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+import urllib3
+from models.address import Address
+from models.listing import Listing
+from urllib3.exceptions import HTTPError
+
+
+class BusinessLicenceClient:
+    BASE_URL = "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/business-licences/records"
+
+    def __init__(self) -> None:
+        self.session = urllib3.PoolManager()
+
+    def _filter_by_licence_number(self, licence_number: str) -> dict:
+        # Omit licence_number validity check since it should be checked during policy evaluation
+        return {"where": f'licencenumber="{licence_number}"'}
+
+    def _filter_by_short_term_rental_business(self) -> dict:
+        return {"where": 'businesstype="Short-Term Rental"'}
+
+    def _licence_status_query(self) -> dict:
+        return {"select": "status"}
+
+    def _make_request(self, fields: dict) -> dict:
+        try:
+            response = self.session.request(
+                method="GET", url=self.BASE_URL, fields=fields
+            )
+            return json.loads(response.data)
+        except HTTPError as e:
+            print(f"Error fetching data: {e}")
+            raise e
+
+    def _process_licence_status_results(self, response_data: dict) -> str | list[str]:
+        total_count = response_data["total_count"]
+        results = response_data["results"]
+        licence_status = [result["status"] for result in results]
+
+        if total_count > 1:
+            print(
+                f"Received {total_count} counts of licence number with statuses: {licence_status}"
+            )
```

Comment:
why is there a print statement here?

Reply:
I just want to emphasize a case where > 2 statuses is returned
Is it ok?

Reply:
are you trying to add a print statement for debugging purposes? if so you can use a logger library. 
if you want to emphasize a case for readers to understand, you can add a comment.

Reply:
I will change it to a comment then